src/main/java/domain/InvoiceDetails.java

```diff
@@ -44,15 +44,15 @@
     private List<Long> locationPointsIds;
 
     private String description;
-    private BigDecimal price;
+    private int price; // Price in cents
     private long distance;
 
     public InvoiceDetails() { }
 
-    public InvoiceDetails(ArrayList<TranslocationDto> locationPoints, String description, BigDecimal price) throws InvoiceException {
+    public InvoiceDetails(ArrayList<TranslocationDto> locationPoints, String description, int categoryPrice) throws InvoiceException {
         if(locationPoints == null || locationPoints.size() < 2) { throw new InvoiceException("No locationPoints provided for InvoiceDetails or only 1 provided."); }
         if(description.isEmpty()) { throw new InvoiceException("Please provide a description for this InvoiceDetail."); }
-        if(price == null || price.compareTo(BigDecimal.ZERO) < 0) { throw new InvoiceException("Please provide a positive price for this InvoiceDetail."); }
+        if(categoryPrice < 1) { throw new InvoiceException("Plesae provide a valid category price"); }
```

Comment:
volgens mij heb ik een check ingebouwd die kijkt of de prijs groter of gelijk aan 0 is. (Misschien rijd de koning gratis ofzo XD)).

Maakt niet uit welke we pakken, maar moeten we even gelijk trekken. Dus allebei 1, of allebei 0.